javascript/src/wombatSetup.js

```diff
@@ -30,6 +30,20 @@ export function urlRewriteFunction(
   // Special stuff which is not really a URI but exists in the wild
   if (['#', '{', '*'].includes(url.substring(0, 1))) return url;
 
+  // If URI scheme is defined but not http or https, we have to not rewrite the URL
+  const uri = URI.parse(url);
+  if (
+    typeof uri.scheme !== 'undefined' &&
+    !['http', 'https'].includes(uri.scheme)
+  )
+    return url;
+
+  // If url starts with prefix, we need to remove this prefix before applying usual
+  // rewrite rules
+  if (url.startsWith(prefix)) {
+    url = uri.scheme + '://' + url.substring(prefix.length);
+  }
```

Comment:
~~Do we really need to apply rewrite rules if url is already rewritten ?~~

We actually need to as we need to properly encode it. Maybe we could separate the encoding from the rewriting (relative path and fuzzy) ?

Reply:
I don't mind to separate these two parts, but I do not consider there will be a great benefit, even if we consider testing. So why not, but not urgent at all (i.e. will probably never be done AFAIK ^^)